languages/python/exercises/concept/chaitanas-colossal-coaster/.meta/config.json

```diff
@@ -8,5 +8,9 @@
       "github_username": "BethanyG",
       "exercism_username": "BethanyG"
     }
-  ]
+  ],
+  "editor": {
+    "solution_files": ["list_methods.py"],
+    "test_files": ["list_methods_test.py"]
```

Comment:
@BethanyG I don't know how the Python test runner/representer/analyzer work, but it _could_ be that they expect these files to use the slug of the exercise. In other words, this _could_ break any of the aforementioned tooling. Might be best to check with someone that knows about the tooling.

Reply:
Yeah, this breaks the current tooling. Can we count on access to this config.json in the test runner etc? If so, it would be better to get filenames from it instead of assuming them.

Reply:
They will be!

Reply:
Then I'll update the tooling later today. For this PR, act as though it won't break tooling.

Reply:
Ok.  Do we want to stick with the stub and test files being named for the concepts, as opposed to the exercise name?

Reply:
Eh... on the one hand, concept-based filenames make for simpler module name for imports. On the other hand, filenames should reflect their contents.. @ErikSchierboom , what are other tracks doing with this?